### ISSUE TITLE
Pagination Control: Focus indicator on the wrong page button

### ISSUE THREAD

**Issue by akeelnazir** [mi6/ic-design-system#1001]
## Summary of the bug

Tell us, in a few words, what the problem is.

## 🪜 How to reproduce

Tell us the steps to reproduce the problem:

1. Go to the Styleguide: '[Pagination](https://design.sis.gov.uk/components/pagination)'
2. Click on: 'Page 5'
3. The user is taken to Page 5, but the focus indicator is on Page 6.

## 📸 Screenshots or code

NA

## 🧐 Expected behaviour

Focus indicator should be on the selected Page 5 when Page 5 button is clicked on the Pagination Control.

**Comment by MI6-255** [comment]
Thanks for your contribution! This issue has been marked as duplication: https://github.com/mi6/ic-ui-kit/issues/1871
